node/sc/kas/anchor_test.go

```diff
@@ -51,9 +51,9 @@ func testAnchorData() *types.AnchoringDataInternalType0 {
 	}
 }
 
-func _TestExampleSendRequest(t *testing.T) {
```

Comment:
What do you want to test in this TC? It seems like a ping test for a dev server.

Reply:
This is an example test code for a dev server.
So I want to except it for common test.

Reply:
You enabled this test in this change.

Reply:
Oops...^^;

Reply:
Disabled. Thanks. :)